etl/steps/data/grapher/dummy/2020-01-01/dummy.py

```diff
@@ -28,7 +24,9 @@ def run(dest_dir: str) -> None:
     # Save outputs.
     #
     # Create a new grapher dataset with the same metadata as the garden dataset.
-    ds_grapher = create_dataset(dest_dir, tables=[tb], default_metadata=ds_garden.metadata)
+    ds_grapher = create_dataset(
+        dest_dir, tables=[tb], default_metadata=ds_garden.metadata, check_variables_metadata=True
+    )
 
     #
     # Checks.
```

Comment:
Shouldn't the checks happen within `create_dataset`? I suppose you will know that you are creating a grapher step from `dest_dir`.

Reply:
Good idea. It started as an experiment, but it appears robust enough to be included there by default.